Source/multi.cpp

```diff
@@ -186,31 +186,23 @@ unsigned char *multi_recv_packet(TBuffer *packet, unsigned char *a2, int *a3)
 	return result;
 }
 
-void multi_send_msg_packet(int a1, BYTE *a2, BYTE len)
+void multi_send_msg_packet(int pmask, BYTE *a2, BYTE len)
 {
-	//const void *v3; // edx
-	signed int v4;   // ebx
-	unsigned int v5; // edi
-	TPkt pkt;        // [esp+Ch] [ebp-204h]
-	int v8;          // [esp+20Ch] [ebp-4h]
+	DWORD v, p, t;
+	TPkt pkt;
 
-	v8 = a1;
 	NetRecvPlrData(&pkt);
-	pkt.hdr.wLen = len + 19;
+	t = len + 19;

```

Comment:
Might be `t = len + sizeof(pkt.hdr)` or `+ sizeof(struct TPktHdr)`.
But that could be adjusted later, I guess.

Reply:
19 is in a lot of places :P I'll leave figuring out what it means to others xD